### ISSUE TITLE
Error when reading a VCF 4.3 file with spaces in INFO field values

### ISSUE THREAD

**Issue by jkmatila** [samtools/htsjdk#1667]
According to the VCF 4.3 specification, spaces are allowed in INFO field values (unlike in VCF 4.2). However, trying to parse such a file fails with an error message "The VCF specification does not allow for whitespace in the INFO field", which seems to come from [AbstractVCFCodec.java:515](https://github.com/samtools/htsjdk/blob/45cfc088608d0efde807f250e9c7d68b19a777a2/src/main/java/htsjdk/variant/vcf/AbstractVCFCodec.java#L515).

See [VCF 4.3 spec](https://samtools.github.io/hts-specs/VCFv4.3.pdf), Section 1.6.1:
> 8. INFO — [...] Space characters are allowed in values.

**Comment by cmnbroad** [comment]
Yes, thanks for reporting this. This is fixed in https://github.com/samtools/htsjdk/pull/1581 (and the commit with the fix is also in https://github.com/samtools/htsjdk/pull/1596).
